sklearn/calibration.py

```diff
@@ -373,6 +389,12 @@ def predict_proba(self, X):
 
         df, idx_pos_class = self._preproc(X)
 
+        proba = self.transform_proba(df, idx_pos_class, n_classes, proba)
```

Comment:
I doubt this should be public

Reply:
Do you mean that this should be called `_transform_proba`?
The problem is that it needs to be called externally form `CalibratedClassiferCV`

I am fine either way - but what is standard policy on calling private functions?